Create the first version of the API (#1)
OpenTermsArchive/federation-api: README.md

# Open Terms Archive Federated API

> Open Terms Archive is a decentralised system that tracks collections of services' terms across multiple servers. Each collection operates its own API which exposes services and terms tracked, but the decentralisation of these APIs implies to search across all these APIs to identify which services and terms are currently tracked. This codebase implements a federation API that allows easy querying of the distributed database, facilitating collaboration with external applications. For a high-level overview of Open Terms Archive’s wider goals and processes, please read its [public homepage](https://opentermsarchive.org).

Reviewer: ```suggestion
Open Terms Archive is a decentralised system that tracks collections of services' terms across multiple servers. Each collection operates its own API which exposes services and terms tracked, but the decentralisation of these APIs implies to search across all of them to identify which and where services and terms are tracked.

This module provides a federated API that allows easy querying of the distributed database, facilitating collaboration with external applications.

For a high-level overview of Open Terms Archive’s wider goals and processes, please read its [public homepage](https://opentermsarchive.org).
```

> For documentation, visit [docs.opentermsarchive.org](https://docs.opentermsarchive.org/)

Reviewer: ```suggestion
For documentation, visit [docs.opentermsarchive.org](https://docs.opentermsarchive.org/).
```
